src/systems/user_commands/UserCommands.cc

```diff
@@ -24,9 +24,11 @@
 #include <ignition/msgs/pose.pb.h>
 #include <ignition/msgs/physics.pb.h>
 #include <ignition/msgs/visual.pb.h>
+#include <ignition/msgs/wheel_slip_parameters.pb.h>
```

Comment:
I think this needs to be updated to `wheel_slip_parameters_cmd.pb.h`

Reply:
Thanks!

I didn't notice the change was needed because after rebuilding the headers with the old name were still available in the install space.